### ISSUE TITLE
Feedback on the scheduler

### ISSUE THREAD

**Issue by malinaa96** [acivitillo/grizly#502]
We would like to collect features that scheduler should have in one place so please share your feedback in this issue.

**Comment by acivitillo** [comment]
- [ ] We need a web UI to manage, display and run jobs

**Comment by krzyzanowskik** [comment]
Well my thoughts:
- Ability to see all jobs [ the simple overview what the job is doing would be awesome too ]
- 'Secured' jobs. Once we agreed that particular job for example 'sales_daily_news' which is making the data load for daily, is correct and working, only Alessio or Kasia will be able to register changes to that job. So we can avoid mistakes that someone will change the job
- Some kind of verification process that job is becoming the secured one.

**Comment by msocha83** [comment]
Like Kamil wrote, a table with all the jobs and their info:
- short explanation what the job is doing, 
- when it's running, is it scheduled for specific time, is it triggered (and by what ideally), 
- owner 
- etc

**Comment by acivitillo** [comment]
@krzyzanowskik and @msocha83 some of those things like "short explanation what the job is doing," will need to be added by you, the end user, to the job. Is that still OK? So the code would look like so

```python
job = Job()
job.description = "short explanation what the job is doing"
```

**Comment by krzyzanowskik** [comment]
Yes of course, we have to just agree (maybe have a template or something) what should be add there.
For instance:
- owner ( in order to know to who raise questions and so on )
- briefly description what the job is doing
- name of final table if applicable 
- if and when it is scheduled
- secured - yes or no

So the explanations will be standardized.

**Comment by malinaa96** [comment]
So this is what we have so far 

```python
        job.owner,
        job.trigger_names,
        job.type,
        job.last_run,
        job.run_time,
        job.status,
        job.error,
        job.created_at,
```
We will add `job.description` but this would be probably one sentence about what the job is doing. I'm not sure how to figure out security but that's a good point.

**Comment by acivitillo** [comment]
Security for now is out of scope we will do in release 2. I would also add `job.assets` to the list so that people know where to find the code for the job and any other resources.

**Comment by michalz92** [comment]
For the future: add a job_run_results that could store output from the job run -- eg. extracts could write previous vs current no. of rows, sums etc. so that we can quickly verify that it ran successfully.

**Comment by acivitillo** [comment]
@malinaa96 can you consolidate any net new request to the top post?

@malinaa96 and @michalz92 can you make sure https://github.com/acivitillo/grizly/milestone/8 is up to date
